repos/system_upgrade/common/actors/opensshconfigscanner/libraries/readopensshconfig.py

```diff
@@ -68,6 +80,22 @@ def parse_config(config):
                 # here we need to record all remaining items as command and arguments
                 ret.subsystem_sftp = ' '.join(el[2:])
 
+        elif el[0].lower() == 'include':
+            # recursively parse the given file or files referenced by this option
+            pattern = el[1]
+            if pattern[0] != '/' and pattern[0] != '~':
+                pattern = os.path.join('/etc/ssh/', pattern)
+            # NOTE that OpenSSH sorts the files lexicographically
+            files = glob.glob(pattern)
+            files.sort()
+            for f in files:
+                output = read_sshd_config(f)
+                if parse_config(output, ret, depth + 1) is None:
```

Comment:
If a function defines a key-value argument, then it can cause future problems not calling it with as a key-value argument. For example, you can add a normaln (non-kwarg) argument and suddenly you break stuff you have believed should have not been broken because the remaining arguments were declared as key-value.
```suggestion
                if parse_config(output, red=ret, depth=depth + 1) is None:
```